python/hail/expr/functions.py

```diff
@@ -382,6 +382,32 @@ def Dict(keys, values):
     ret_type = TDict(key_col._type, value_col._type)
     return _func("Dict", ret_type, keys, values)
 
+@typecheck(x=expr_numeric, a=expr_numeric, b=expr_numeric)
+def dbeta(x, a, b):
+    """
+    Returns the probability density at x of a `beta distribution <https://en.wikipedia.org/wiki/Beta_distribution>`__
+    with parameters a (alpha) and b (beta).
+
```

Comment:
Sorry I didn't notice this earlier. All of the other expr functions have examples. Can you put those in for both `dbeta` and `plDosage` ?